src/github/index.js

```diff
@@ -21,15 +20,25 @@ const GITHUB_REPO_URL = 'https://github.com/ambanum/OpenTermsArchive/blob/master
 const GOOGLE_URL = 'https://www.google.com/search?q=';
 
 export default class GitHub {
+  static isValidToken() {
+    return GITHUB_REPO.includes('/');
+  }
+
   constructor() {
+    if (!GitHub.isValidToken('')) {
```

Comment:
Why passing empty string in param?

Reply:
By the way, this condition will never be true, as you test the token before instantiating GitHub module:
```
  if (GitHub.isValidToken()) {
    app.attach(new GitHub());
  }
  ```

Reply:
1. empty param is wrong, i'll remove it
2. I think it will as it's static